packages-openshift.yaml

```diff
@@ -1,3 +1,9 @@
+metadata:
+  # This should match the /etc/os-release manipulation we do below when
+  # injecting `OPENSHIFT_VERSION`. It's used by CI to determine the repos to
+  # inject when building the layered image.
+  ocp_version: "4.18"
```

Comment:
Aren't we 4.19 now?

Reply:
Yeah, this will be bumped along with the other 4.18 references in the repo.